app/assets/javascripts/helpers/prop_types.jsx

```diff
@@ -1,40 +1,39 @@
 import React from 'react';
+window.shared || (window.shared = {});
```

Comment:
Can `window.shared` be removed here too?

Reply:
I will remove them. I wasn't sure if I should keep it either. The test runs well without it and the files were building in yarn start.